tests/sweep_framework/sweeps/eltwise/unary/sigmoid_accurate/sigmoid_accurate.py

```diff
@@ -41,22 +56,44 @@
 # The runner will call this run function with each test vector, and the returned results from this function will be stored.
 # If you defined a device_mesh_fixture above, the object you yielded will be passed into this function as 'device'. Otherwise, it will be the default ttnn device opened by the infra.
 def run(
-    input_shape,
-    input_a_dtype,
-    input_a_layout,
-    input_a_memory_config,
-    output_memory_config,
+    input_shape=None,
+    input_a_dtype=None,
+    input_a_layout=None,
+    input_a_memory_config=None,
+    output_memory_config=None,
+    traced_config_name=None,
     *,
     device,
 ) -> list:
+    # Handle traced_config_name parameter (for model_traced suite)
+    # Use the helper to unpack all config values in one line
+    if traced_config_name is not None:
+        input_shape, input_a_dtype, input_a_layout, input_a_memory_config, output_memory_config = unpack_traced_config(
+            traced_config_name
+        )
+
+    # Validate and set defaults for other suites
+    if input_shape is None:
+        raise ValueError("input_shape must be provided")
+    if input_a_dtype is None:
+        input_a_dtype = ttnn.bfloat16
+    if input_a_layout is None:
+        input_a_layout = ttnn.TILE_LAYOUT
+    if input_a_memory_config is None:
+        input_a_memory_config = ttnn.DRAM_MEMORY_CONFIG
+    if output_memory_config is None:
+        output_memory_config = ttnn.DRAM_MEMORY_CONFIG
```

Comment:
If any of these is ever true, this suggests a bug in `MasterConfigLoader.get_suite_parameters`, would suggest removing this validation block and the default `=None` from the parameters above (and in the previous file), and addressing any issues (assuming there are any) in the implementation of `get_suite_parameters` rather than requiring checks in `run`.

Alternatively, remove this validation block and replace the `=None` default parameters above with the respective defaults you substitute in the validation block here (omitting a default argument for `input_shape` since none is provided), if that's really the expected usage of `MasterConfigLoader`.